cmd/ova-provider-server/ova-provider-server.go

```diff
@@ -299,28 +301,56 @@ func findOVAFiles(directory string) ([]string, []string, error) {
 	}
 
 	var ovaFiles, ovfFiles []string
+
+	// Check for .ova files in the main directory (zero level)
+	for _, child := range childs {
+		path := filepath.Join(directory, child.Name())
+		if hasSuffixIgnoreCase(child.Name(), OvaExt) {
+			ovaFiles = append(ovaFiles, path)
+		}
+	}
```

Comment:
why we need this loop while having the same loop below? can't we drop the `if !child.IsDir() {` and deal with this case?

Reply:
+1